guacamole-docker/README.md

```diff
@@ -115,6 +119,35 @@ The process for doing this via the `mysql` utility included with MySQL is
 documented in
 [the Guacamole manual](http://guacamole.incubator.apache.org/doc/gug/jdbc-auth.html#jdbc-auth-mysql).
 
+
+Deploying Aoache Guacamole with LDAP authentication
+--------------------------------------------------
+
+    docker run --name some-guacamole --link some-guacd:guacd    \
+        -e LDAP_HOSTNAME=172.17.42.1                            \
+        -e LDAP_USER_BASE_DN=ou=people,dc=example,dc=com        \
+        -e LDAP_CONFIG_BASE_DN=ou=connections,dc=example,dc=com \
+        -d -p 8080:8080 glyptodon/guacamole
+
+Linking Guacamole to your LDAP directory will require additional configuration parameters
```

Comment:
I think technically it's not "linking" in this case, since Docker links are not involved (no `--link`).